apps/services/user-notification/src/app/modules/notifications/notificationsWorker.service.ts

```diff
@@ -29,6 +51,185 @@ export class NotificationsWorkerService implements OnApplicationBootstrap {
     }
   }
 
+  async handleDocumentNotification({
+    profile,
+    messageId,
+    message,
+  }: HandleNotification) {
+    // don't send message unless user wants this type of notification
+    if (!profile.documentNotifications) {
+      this.logger.info(
+        'User does not have notifications enabled this message type',
+        { messageId },
+      )
+
+      return
+    }
+
+    this.logger.info('User has notifications enabled this message type', {
+      messageId,
+    })
+
+    const notification = await this.messageProcessor.convertToNotification(
+      message,
+      profile,
+    )
+
+    await this.notificationDispatch.sendPushNotification({
+      nationalId: profile.nationalId,
+      notification,
+      messageId,
+    })
+  }
+
+  createEmail({
+    isEnglish,
+    profile,
+    template,
+    formattedTemplate,
+    fullName,
+  }: {
+    isEnglish: boolean
+    profile: UserProfile
+    template: HnippTemplate
+    formattedTemplate: HnippTemplate
+    fullName: string
+  }): Message {
+    return {
+      from: {
+        name: 'Ísland.is',
+        address: 'no-reply@island.is',
+      },
+      to: {
+        name: fullName,
+        address: profile.email,
+      },
+      subject: template.notificationTitle,
+      template: {
+        title: 'Testinga',
+        body: [
```

Comment:
We need to sync offline about the email body on monday as we need to support all the active push notifications.